src/stylesheets/book.css

```diff
@@ -0,0 +1,174 @@
+.books-container {
+  width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ] It would be better if we fix the DRY principle, the `align-items: center`, `display: flex` have been repeated. Let us kindly address it. For example; ``` .class1, .class2 { display: flex; align-items: center; }```

Reply:
I am totally and absolutely confused, for me personally and I would still need to edit each classes separately its why I cannot follow the principle all the time.

Reply:
@AkintoyeOlamide, yeah I understand, at least lesson learned, you did change some, next let's build our CSS using good practice.